src/soter/openssl/soter_asym_ka.c

```diff
@@ -168,7 +168,7 @@ soter_status_t soter_asym_ka_import_key(soter_asym_ka_t* asym_ka_ctx, const void
     /*
      * soter_ec_{priv,pub}_key_to_engine_specific() expect EVP_PKEY of EVP_PKEY_EC type
      * to be already allocated and non-NULL. We might be importing it anew, or we might be
-     * replacing previously generated keypair.
+     * replacing previously generated key pair.
```

Comment:
Okay. This has been a historical point of contention, now I think is a good time to make a decision.

I propose to bless “key pair” as the preferred spelling in prose, and `KeyPair` or `key_pair` in code.

Rationale: this spelling is used in JP's *Serious Cryptography*.